### DISCUSSION TITLE
How many colors does OM support?

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#1172] [Q&A]
Hello,

According to its (now gone) FAQ, Maps.me supported the following colors for waypoints/tracks: yellow, blue, green, purple, orange, brown, pink.

But I notice OM supports more:
![https://i.postimg.cc/yNZMqFRg/image.png](https://i.postimg.cc/yNZMqFRg/image.png)

What gives?

Thank you.

**Comment by biodranik** [comment]
You see all currently supported colors on the screenshot :)

**Reply by Shohreh** [reply]
Cool.

Where's the list, so I know how to edit KML files accordingly?

#Used for wp in route red – ff0000ff
yellow – ff00ffff
blue – ffff0000
green – ff00ff00
purple – ff800080
orange – ff0080ff
brown – ff336699
pink – ffff00ff

**Comment by biodranik** [comment]
{"placemark-red", "BookmarkRed"},
  {"placemark-blue", "BookmarkBlue"},
  {"placemark-purple", "BookmarkPurple"},
  {"placemark-yellow", "BookmarkYellow"},
  {"placemark-pink", "BookmarkPink"},
  {"placemark-brown", "BookmarkBrown"},
  {"placemark-green", "BookmarkGreen"},
  {"placemark-orange", "BookmarkOrange"},
  {"placemark-deeppurple", "BookmarkDeepPurple"},
  {"placemark-lightblue", "BookmarkLightBlue"},
  {"placemark-cyan", "BookmarkCyan"},
  {"placemark-teal", "BookmarkTeal"},
  {"placemark-lime", "BookmarkLime"},
  {"placemark-deeporange", "BookmarkDeepOrange"},
  {"placemark-gray", "BookmarkGray"},
  {"placemark-bluegray", "BookmarkBlueGray"}

**Reply by Scrapporoid** [reply]
biodranik,

What is the base Organic Maps URL for the bookmarks to put in the <href> tags? E.G., for Maps.me, you would have "http://mapswith.me/placemarks/placemark-green.png" and change out the "placemark-green.png" for other colors.

Thanks,
Shawn

**Reply by biodranik** [reply]
https://omaps.app/placemarks/placemark-pink.png

**Reply by Scrapporoid** [reply]
Thank you!

**Comment by biodranik** [comment]
106:  BookmarkRed-color: #E51B23;
107:  BookmarkPink-color: #FF4182;
108:  BookmarkPurple-color: #9B24B2;
109:  BookmarkDeepPurple-color: #6639BF;
110:  BookmarkBlue-color: #0066CC;
111:  BookmarkLightBlue-color: #249CF2;
112:  BookmarkCyan-color: #14BECD;
113:  BookmarkTeal-color: #00A58C;
114:  BookmarkGreen-color: #3C8C3C;
115:  BookmarkLime-color: #93BF39;
116:  BookmarkYellow-color: #FFC800;
117:  BookmarkOrange-color: #FF9600;
118:  BookmarkDeepOrange-color: #F06432;
119:  BookmarkBrown-color: #804633;
120:  BookmarkGray-color: #737373;
121:  BookmarkBlueGray-color: #597380;
